src/js/components/Video/stories/typescript/TheaterMode.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import { Box, Grommet, Video, grommet } from 'grommet';
+import { Monitor } from 'grommet-icons/icons/Monitor';
+
+const TheaterMode = (props) => {
+  const [source, setSource] = useState('small_video.mp4');
+
+  return (
+    <Grommet theme={grommet}>
+      <Box
+        background={source === 'small_video.mp4' ? '' : 'dark-1'}
+        overflow="hidden"
+        height={{ max: 'large' }}
+        align="center"
+        pad="large"
+      >
+        <Video
+          src={source}
+          fit="cover"
+          controls={{
+            position: 'below',
+            items: [
+              'volume',
+              'reduceVolume',
+              {
+                icon: <Monitor />,
+                onClick: () =>
+                  setSource(
```

Comment:
I'm wondering if leaving the source alone but changing the Box's dimensions might be a better approach, and potentially cause less flicker when toggling the mode. Maybe start with the Box having 'small' width and then unsetting `width` and setting `fill="horizontal"` for theater mode. Can you try that?